lib/themes/dosomething/paraneue_dosomething/js/components/Slideshow.js

```diff
@@ -24,30 +24,39 @@ class Slideshow extends React.Component {
     window.scrollTo(0, 0);
   }
 
-  moveBackward() {
-    const slide = this.state.currentSlide - 1;
-    sendEvent('Onboarding (Prev btn)', 'Slide Change', slide);
+  getCurrentSlideName() {
+    // Create a dummy component so we can grab the name.
+    // TODO: This isn't the ideal way to do this, but this PR is contained to just data improvements.
+    return React.createElement(this.props.slides[this.state.currentSlide], {key: 1, campaign: {}, user: {}, isActive: false}).type.name;
```

Comment:
Maybe just add some more info on exactly _why_ you need to do this and create a dummy element, so if someone else jumps in to clean up they know what's going on...? I'm defs a bit confused.